docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -102,31 +102,25 @@ Containers:
 
 ### When should you use liveness or readiness probes?
 
-If the process in your Container is able to crash on its own whenever it
-encounters an issue or becomes unhealthy, you do not necessarily need a liveness
-probe; the kubelet will automatically perform the correct action in accordance
-with the Pod's `restartPolicy`.
-
-If you'd like your Container to be killed and restarted if a probe fails, then
-specify a liveness probe, and specify a `restartPolicy` of Always or OnFailure.
-
-If you'd like to start sending traffic to a Pod only when a probe succeeds,
-specify a readiness probe. In this case, the readiness probe might be the same
-as the liveness probe, but the existence of the readiness probe in the spec means
-that the Pod will start without receiving any traffic and only start receiving
-traffic after the probe starts succeeding.
-
-If your Container needs to work on loading large data, configuration files, or migrations during startup, specify a readiness probe.
-
-If you want your Container to be able to take itself down for maintenance, you
-can specify a readiness probe that checks an endpoint specific to readiness that
-is different from the liveness probe.
-
-Note that if you just want to be able to drain requests when the Pod is deleted,
-you do not necessarily need a readiness probe; on deletion, the Pod automatically
-puts itself into an unready state regardless of whether the readiness probe exists.
-The Pod remains in the unready state while it waits for the Containers in the Pod
-to stop.
+No matter how great readiness and liveness probes can be, they’re not always necessary. Here are some use cases of liveness and readiness probes.
+
+#### You possibly need liveness probe if:
+
+- you'd like your Container to be killed and restarted if a probe fails, then specify a liveness probe, and specify a `restartPolicy` of Always or OnFailure.
+
+#### You don't necessarily need liveness probe if:
+
+- the process in your Container is able to crash on its own whenever it encounters an issue or becomes unhealthy. In this case, you do not necessarily need a liveness probe since the kubelet will automatically perform the correct action in accordance with the Pod's `restartPolicy`.
+
+#### You possibly need readiness probe if:
+
+- you'd like to start sending traffic to a Pod only when a probe succeeds. In this case, the readiness probe might be the same as the liveness probe, but the existence of the readiness probe in the spec means that the Pod will start without receiving any traffic and only start receiving traffic after the readiness probe starts succeeding.
```

Comment:
Saying the two probes might be the same might be confusing to people. It might be better to pull in the next point (about needing time to prep stuff before it’s ready to process incoming traffic) into this bullet and talk about the diff between the two rather than hinting that they might be the same.